app/templates/createquizsuccess.html

```diff
@@ -0,0 +1,56 @@
+﻿{% extends "base.html" %}
+{% block style %}
+<link rel="stylesheet" href="../static/content/createstyle.css">
+{% endblock %}
+{% block content %}
+<div class="create-class-wrapper">
+    <div class="create-container">
+
+        <form action="" method="post" novalidate class="create-form">
+            {{ form.hidden_tag() }}
+            <h1>Create A New Quiz</h1>
+            {{ form.topic(placeholder="Select Topic", class_="center") }}
+            {{ form.qn(size=32, placeholder="Input Question") }}
+            {{ form.op1(size=32, placeholder="Option 1") }}
+            {{ form.op2(size=32, placeholder="Option 2") }}
+            {{ form.op3(size=32, placeholder="Option 3") }}
+            {{ form.op4(size=32, placeholder="Option 4") }}
+            {{ form.corrOp(placeholder="Correct Option") }}
+            {{ form.img }}
+            <a href="{{ url_for('createquiz') }}" class="route2-btn">Add Another Question</a>
+            <a href="{{ url_for('createquizsuccess') }}" class="route2-btn">Submit Quiz</a>
```

Comment:
Why does the `createquizsuccess` page have the options for creating a question?

Reply:
I created a modal popup, let me show you 
![image](https://user-images.githubusercontent.com/54029207/84887155-f3979700-b0c7-11ea-98e7-5f6307187264.png)